hoodie-client/src/main/java/com/uber/hoodie/io/HoodieCommitArchiveLog.java

```diff
@@ -126,9 +127,11 @@ public boolean archiveIfRequired() {
     HoodieTable table = HoodieTable.getHoodieTable(metaClient, config);
 
     // GroupBy each action and limit each action timeline to maxCommitsToKeep
+    // TODO: Handle ROLLBACK_ACTION in future
```

Comment:
Let's add ROLLBACK_ACTION in the HoodieActiveTimeline and see if there are any repurcussions ?

Reply:
This would need more tests especially internally in Uber given the amount of historical rollback actions.  So I'm a bit hesitate to do this in this PR and prefer to do this later.

Reply:
lets open an issue for this..

Reply:
https://github.com/uber/hudi/issues/334 opened for this.  And https://github.com/uber/hudi/issues/335 is opened for handling inflight instants.